src/command_modules/azure-cli-container/setup.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+    cmdclass = {}
+
+VERSION = '2.0.10b1'
```

Comment:
Change the version to not include `b1`. Some older versions of pip would not install this package as it's considered a prerelease with the `b1`.

Currently, in cloud shell, you have 'container (0.1.5b2+dev)'. So consider making this version 0.1.6+dev.